app/components/rdfa-editor-container.hbs

```diff
@@ -104,19 +104,6 @@
                 {{yield to='sidebar'}}
                 {{#if @shouldEditRdfa}}
                   <Plugins::Link::LinkEditor @controller={{this.controller}} />
```

Comment:
I don't really know how this got in here, but since we're pretty settled on the popout link editor, we can remove this sidebar version here